zerver/tests/test_subs.py

```diff
@@ -2014,7 +2015,7 @@ def test_multi_user_subscription(self) -> None:
             set([user1.id, user2.id, self.test_user.id])
         )
 
-        self.assertEqual(len(add_peer_event['users']), 19)
+        self.assertEqual(len(add_peer_event['users']), 18)
```

Comment:
My reasoning behind this is that Guest user isn't sent an event that's why this is decreased.